src/web-components/comments/components/types.ts

```diff
@@ -13,6 +13,11 @@ export enum CommentMode {
   READONLY = 'readonly',
 }
 
+export enum PinMode {
+  ADD = 'add',
+  SHOW = 'SHOW',
+}
```

Comment:
Why is 'SHOW' in uppercase while 'add' is in lowercase? Wouldn't it be better to use all lowercase?
```suggestion
export enum PinMode {
  ADD = 'add',
  SHOW = 'show',
}
```